criu/cr-restore.c

```diff
@@ -76,6 +75,8 @@
 #include "sigframe.h"
 #include "fdstore.h"
 #include "string.h"
+#include "memfd.h"
+#include "string.h"
```

Comment:
Again include statement duplicate. We already have bc49927bb ("criu: Make use strlcpy() to copy into allocated strings"). More over we have ```#include <string.h>``` three times in this c file.